src/azure-cli/azure/cli/command_modules/storage/_help.py

```diff
@@ -566,10 +584,12 @@
 short-summary: Create the url to access a blob.
 examples:
   - name: Create the url to access a blob (autogenerated)
-    text: az storage blob url --connection-string $connectionString --container-name container1 --name blob1
+    text: |
+        az storage blob url --connection-string $connectionString --container-name container1 --name blob1
     crafted: true
   - name: Create the url to access a blob (autogenerated)
-    text: az storage blob url --account-name storageacct --container-name container1 --name blob1
+    text: |
+        az storage blob url --account-name storageacct --container-name container1 --name blob1
```

Comment:
duplicated. only difference is with and without --connection-string but the example name is the same.

Reply:
This is what Aladdin said "not all parameters affect the title of the example, so when we generate more than one example for a command it’s quite likely to see repeated title".

I guess Aladdin is trying to tell user the likely way to use this command

Reply:
Let's accept it this time and hope they can refine example names next time.